### PR TITLE
Update conjur-authn-k8s-client version to 0.18.1

### PR DESCRIPTION
**Pull request by abrahamko**
### What does this PR do?
Update conjur-authn-k8s-client version to 0.18.1

### What ticket does this PR close?
Connected to #223

#### Change log
- [x] The CHANGELOG has been updated, or

#### Test coverage
- [x] The changes in this PR do not require tests

#### Documentation
- [x] This PR does not require updating any documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
-## [1.1.0] - 2020-08-20
```
Comment (orenbm) [inline CHANGELOG.md:9]: @abrahamko why was this removed?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,14 +5,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
 ### Added
 - Helm chart to deploy the Secrets Provider Job using Helm ([cyberark/secrets-provider-for-k8s#165](https://github.com/cyberark/secrets-provider-for-k8s/pulls/165))
 - CONTAINER_MODE support for application containers in addition to init containers ([cyberark/secrets-provider-for-k8s#179](https://github.com/cyberark/secrets-provider-for-k8s/pull/179))
 - Red Hat certified version of the secrets-provider-for-k8s image ([cyberark/secrets-provider-for-k8s#93](https://github.com/cyberark/secrets-provider-for-k8s/pull/93))
 
 ### Changed
+- Bumped the authn-k8s client version to 0.18.1 
+  [cyberark/conjur-authn-k8s-client#223](https://github.com/cyberark/conjur-authn-k8s-client/issues/223)
```
Comment (orenbm) [inline CHANGELOG.md:15]: this should be under `Unreleased`. if i understand correctly, we already released version `1.1.0` with the changes here.

**Comment by orenbm** [comment]
@abrahamko can you explain why we need this version?

File: CHANGELOG.md (hunk 1 above)
Comment (sigalsax) [reply CHANGELOG.md:9]: because we haven't released yet @orenbm

File: CHANGELOG.md (hunk 2 above)
Comment (sigalsax) [reply CHANGELOG.md:15]: we didn't actually release because of a pipeline failure so though the changes were made, the release never happened @orenbm

**Comment by abrahamko** [comment]
@orenbm We don't have to use this version, but @Tovli suggested to use the 0.18.1 version you released yesterday in secrets provider too, and I looked into it and it seems like a good idea.
Especially for the benefit of waiting to the cert file to arrive.

**Review by orenbm — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (sigalsax) [reply CHANGELOG.md:15]: should follow the following format: ([cyberark/conjur-authn-k8s-client#223](https://github.com/cyberark/conjur-authn-k8s-client/issues/223))
